package/kubectl/entrypoint.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash -x
+
+SSL_CRTS_DIR=/etc/kubernetes/ssl
+mkdir -p $SSL_CRTS_DIR
+
+for i in $(env | grep -o KUBE_.*=); do
```

Comment:
i don't think this part is needed because the admin kubeconfig should include the certificate data